applications/nrf_desktop/src/modules/hid_state.c

```diff
@@ -1046,6 +1044,7 @@ static void init(void)
 	if (IS_ENABLED(CONFIG_DESKTOP_HID_REPORT_MOUSE_SUPPORT)) {
 		report_index[REPORT_ID_MOUSE] = id;
 		state.report_data[id].items.item_count_max = MOUSE_REPORT_BUTTON_COUNT_MAX;
+		state.report_data[id].axes.axis_count = 3; /* X, Y, Wheel */
```

Comment:
Is this used anywhere?

Reply:
not at the moment

Reply:
So maybe we can remove it to save memory? We know how big the array is and we may make checks based on size of the array.